examples/mobile/ScheduleApp/js/app.js

```diff
@@ -4,30 +4,47 @@
 			this._store = [];
 			this._ui = {}
 		},
+		tempDate = new Date(),
+		defaultStore = [{
+			on: tempDate.getTime(),
+			off: tempDate.getTime() + 3800 * 1000, // + 1h
```

Comment:
Isn't it 3600 in an hour?

Reply:
Thanks. I modified. I didn't want a full hour to see the difference more clearly